docs/learn/platformio.md

```diff
@@ -0,0 +1,79 @@
+
+![!VS Code Waiting](https://www.v1engineering.com/wp-content/uploads/2019/12/Still_installing.jpg){: width="400"}
+
+At the very bottom of the VS Code window you will see a purple bar and the installing icon and message, touch nothing while it works. When it is done it will pop up a restart button.
+
+![!VS Code restart](https://www.v1engineering.com/wp-content/uploads/2019/12/restart.jpg){: width="400"}
+
+Congratulations you are done with the "hard part"!
+
+---
+
+## Using PlatformIO
+
+Once you have PlatformIO installed there is really nothing else that needs to be added. Now you can load your firmware of choice and flash or compile.
+
+### Load Your Firmware
+
+You should already have downloaded and **Unzipped** the firmware for your board. If you are unsure of that step have a look [here](../electronics/marlin-firmware.md).
+
+In the top left of the VS Code window select "File", then "Open Folder".
+
+![!VS Code Open Folder](https://www.v1engineering.com/wp-content/uploads/2019/12/open-folder.png){: width="400"}
+
+You can select the entire folder you have unzipped. If that is unclear - You need the entire folder that contains "platformio.ini" and the "marlin" folder. Here is a picture of what that looks like. The name will vary as the versions progress, bugfix 2.0.x is another common one.
+
+![!VS Code Folder](https://www.v1engineering.com/wp-content/uploads/2019/12/folder-selected.jpg){: width="400"}
+
+One you open that folder it will show in the window and look very similar to the picture below. **Notice** at the very bottom, the blue banner shows a "Index Rebuild" icon let that finish before moving to the next step.
+
+![!VS Code Folder Open](https://www.v1engineering.com/wp-content/uploads/2019/12/folder-open.jpg){: width="400"}
+
+### Build
+
+At this point it is ready to go. Most boards can be plugged in a "flashed"  or some boards just want you to compile the firmware and load it onto an SD card. Only the last step differs.
+
+Hit the check mark to build (or compile) the firmware. Most items will say "ignored" as it only builds for the one specified board.
+
+![!VS Code Compile](https://www.v1engineering.com/wp-content/uploads/2019/12/build.png){: width="400"}
+
+At this point you should get a message that says "1 succeeded". If you do not the firmwre has build issues and they need to be resolved. 
+
+The next step is to get it onto you board, or if you are sharing your firmware you can skip to the Compile step.
+
+**Flashing** (most common)
+
+You do not need to select a com port as in Arduino, just plug in your board and it should be recognized even USB card based boards. Then and hit the right arrow (1) button and wait for it to transfer.
+
+![!VS Code Flash](https://www.v1engineering.com/wp-content/uploads/2019/12/uploaded_done.png){: width="400"}
+
+When you see (2) "1 succeeded", and you are DONE and the board is ready to use!! If you do not see that there are board connection issues.
+
+
+**Compile to file**
```

Comment:
The word "compile" here is tripping my OCD. It compiled it as part of the build step (and then, it linked it, or archived it, creating the bin).

I would change this title to "Saving the Binary" or "Saving the Firmware Binary".